models/intel/text-spotting-0002-recognizer-decoder/description/text-spotting-0002-recognizer-decoder.md

```diff
@@ -33,14 +32,13 @@ This model is 2D attention-based GRU decoder of text recognition head.
 ## Inputs
 
 1.	Name: `encoder_outputs` , shape: [1x(28*28)x256]. Encoded text recognition features.
-1.	Name: `prev_symbol` , shape: [1x1]. Index in alphabet of previously generated symbol.
-1.	Name: `prev_hidden`, shape: [1x1x256]. Previous hidden state of GRU.
+2.	Name: `prev_symbol` , shape: [1x1]. Index in alphabet of a previously generated symbol.
```

Comment:
I believe it's _the_ previously generated symbol. You apply this model to get one symbol at a time, each time passing the symbol that was decoded on the previous iteration.